README.md

```diff
@@ -121,22 +108,41 @@ Example:
   
 - `adContainerId`  
   **Required** String. Id of element in the DOM where an ad will be rendered.
-  
-## Run unit tests
+
+
+## Contributing
+
+### Install
```

Comment:
Now "Install" and "Build" sections are part of "Contributing" chapter.
I am not sure it would be clear for users how to get library source file. (except those users that are going contribute it).